zerver/openapi/zulip.yaml

```diff
@@ -15650,26 +15650,6 @@ paths:
                               **Changes**: New in Zulip 10.0 (feature level 288). Previously, this
                               permission was controlled by `delete_own_message_policy`.
                           - $ref: "#/components/schemas/GroupSettingValue"
-                      realm_delete_own_message_policy:
-                        type: integer
-                        description: |
-                          Present if `realm` is present in `fetch_event_types`.
-
-                          The [policy][permission-level] defining which users can delete
-                          messages that they had sent.
-
-                          - 1 = Members only
-                          - 2 = Admins only
-                          - 3 = [Full members][calc-full-member] only
-                          - 4 = Admins and moderators only
-                          - 5 = Everyone
-
-                          **Changes**: New in Zulip 5.0 (feature level 101), replacing the
-                          previous `allow_message_deleting` boolean;
-                          `true` corresponded to `everyone`, and `false` to `admins only`.
-
-                          [permission-level]: /api/roles-and-permissions#permission-levels
-                          [calc-full-member]: /api/roles-and-permissions#determining-if-a-user-is-a-full-member
```

Comment:
We likely should preserve this change history in the new setting's documentation.

Reply:
Added a changes entry:
```
Before Zulip 5.0 (feature level 101), the `allow_message_deleting` boolean
was used, where `true` corresponded to `everyone`, and `false` to `admins only`.
```

Reply:
Great. I also added the actual mapping, so one can write code to work with both old and new versions without needing to spelunk the commit history.